base/server/python/pki/server/__init__.py

```diff
@@ -1302,7 +1303,7 @@ def delete_external_cert(self, nickname, token):
         self.save_external_cert_data()
 
     def save_external_cert_data(self):
-        with io.open(self.external_certs_conf, 'wb') as f:
+        with io.open(self.external_certs_conf, 'w') as f:
```

Comment:
Replace with `with open`? They both mean the same. 

Ref: https://www.reddit.com/r/learnpython/comments/7wnldj/ioopen_vs_open/

Reply:
Okay. I'll update it too.